### PR TITLE
Create membership.md

### PR DESCRIPTION
**Pull request by DavidPHirsch**
Signed-off-by:David Hirsch <David.hirsch@dynatrace.com>

## This PR
creating an ISSUE template for membership

### CODE REVIEW

File: ISSUE_TEMPLATE/membership.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in a OpenFeature Org
```
Comment (AlexsJones) [inline ISSUE_TEMPLATE/membership.md:3]: This doesn't seem grammatically correct perhaps "Request membership in the OpenFeature Org

File: ISSUE_TEMPLATE/membership.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Sponsors
+
+<!-- Replace (at) with the `@` sign -->
+
+- (at)sponsor-1
+- (at)sponsor-2
+
+Each sponsor should reply to this issue with the comment "*I support*".
+Please remember, it is an applicant's responsibility to get their sponsors' confirmation before submitting the request.
```
Comment (AlexsJones) [inline ISSUE_TEMPLATE/membership.md:34]: I don't think verbs use apostrophes `sponsors` is correct I believe

**Review by AlexsJones — APPROVED** [review]

**Review by AlexsJones — DISMISSED** [review]
comments inlined

File: ISSUE_TEMPLATE/membership.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in the OpenFeature Org
+title: 'REQUEST: New membership for <your-GH-handle>'
+labels: area/github-membership
+assignees: ''
```
Comment (beeme1mr) [inline ISSUE_TEMPLATE/membership.md:6]: Do you know if it's possible to set a group as a default assignee? If so, I think we should use `@open-feature/maintainers`.

Comment (DavidPHirsch) [reply ISSUE_TEMPLATE/membership.md:6]: I think it is a setting we can add after

Comment (AlexsJones) [reply ISSUE_TEMPLATE/membership.md:6]: It is possible yes in the CODEOWNERS file

Comment (DavidPHirsch) [reply ISSUE_TEMPLATE/membership.md:6]: ok then if we approve this, we can add the details

File: ISSUE_TEMPLATE/membership.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<!-- Please remember to change the title of this issue by replacing
+ <your-GH-handle> with the actual GitHub handle -->
+
+### GitHub Username
+
+e.g. (at)example_user
+
+### Requirements
+
+- [ ] I have reviewed the community membership guidelines ADD LINK
```
Comment (beeme1mr) [inline ISSUE_TEMPLATE/membership.md:19]: Add a link to the guidelines

Comment (DavidPHirsch) [reply ISSUE_TEMPLATE/membership.md:19]: we need to first publish the contributor ladder

File: ISSUE_TEMPLATE/membership.md (hunk 3 above)
Comment (DavidPHirsch) [reply ISSUE_TEMPLATE/membership.md:6]: can be done in the settings once we have created the template

File: ISSUE_TEMPLATE/membership.md (hunk 2 above)
Comment (DavidPHirsch) [reply ISSUE_TEMPLATE/membership.md]: here is would be the confirmation of their sponsor

**Review by DavidPHirsch — COMMENTED** [review]
correction and edit made

**Review by beeme1mr — APPROVED** [review]

**Review by toddbaert — APPROVED** [review]

**Comment by beeme1mr** [comment]
@AlexsJones do you have any additional concerns?

**Pull request merged**